src/adapters/arpalazio.js

```diff
@@ -113,7 +113,7 @@ export const getStream = function (cityName, url, averagingPeriod, source, orgUr
   const unit = getUnit(parameter);
   const dayPosition = averagingPeriod.value === 1 ? 0 : 1;
 
-  const fewDaysAgo = +Number(moment.tz(timezone).subtract(4, 'days').format('DDD'));
+  const fewDaysAgo = +parseFloat(DateTime.local().setZone(timezone).minus({days: 4}).ordinal);
```

Comment:
Why do we check 4 days ago?

Reply:
I will create an issue to audit this adapter.